ihmc-high-level-behaviors/src/main/java/us/ihmc/behaviors/sequence/actions/FootstepPlanActionPlanningThread.java

```diff
@@ -123,13 +125,32 @@ else if (definition.getPlannerInitialStanceSide().getValue() == InitialStanceSid
 
       if (!isPreviewPlanner)
          state.getLogger().info("Planning footsteps...");
+
       FootstepPlannerOutput footstepPlannerOutput = footstepPlanner.handleRequest(footstepPlannerRequest, isPreviewPlanner);
-      FootstepPlan footstepPlan = footstepPlannerOutput == null ? new FootstepPlan() : footstepPlannerOutput.getFootstepPlan();
-      if (!isPreviewPlanner)
-         state.getLogger().info("Footstep planner completed with {}, {} step(s)", footstepPlannerOutput.getFootstepPlanningResult(), footstepPlan.getNumberOfSteps());
+      boolean plannerBusy = footstepPlannerOutput == null;
+      boolean foundSolution = !plannerBusy && footstepPlannerOutput.getFootstepPlanningResult() == FootstepPlanningResult.FOUND_SOLUTION;
```

Comment:
Importantly, we check if the result is FOUND_SOLUTION instead of having > 0 steps. I don't think we want to execute the plan unless fully successful.